net-irc/inspircd/inspircd-3.11.0.ebuild

```diff
@@ -0,0 +1,122 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
```

Comment:
8 should work here. I'll test and add it if it does. 

Thanks!

Reply:
Thanks juippis!